Net24Solution/Net24Task/WebSite.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.NetworkInformation;
+using System.Net.Mail;
+using System.Net;
+using System.Dynamic;
+using System.Timers;
+
+namespace Net24Task
+{
+    public class WebSite
+    {
+        public int CheckInterval { get; set; }
+        public int ServerResponseTime { get; set; }
+        public string PageUrl { get; set; }
+        public string AdminEmail { get; set; }
+
+        public Ping PingSite { get; set; }
+        public PingReply SiteReply { get; set; }
```

Comment:
`public int CheckInterval { get; set; }
        public int ServerResponseTime { get; set; }
        public string PageUrl { get; set; }
        public string AdminEmail { get; set; }

        public Ping PingSite { get; set; }
        public PingReply SiteReply { get; set; }`

convert properties to private fields